proc_stat_io.go

```diff
@@ -0,0 +1,54 @@
+package procfs
+
+import (
+	"fmt"
+	"io/ioutil"
+)
+
+// ProcIO models the content of /proc/<pid>/io.
+type ProcIO struct {
+	// Chars read.
+	Rchar uint64
```

Comment:
I'd go for `RChar`, `WChar`, `SyscR`, ... (compare with existing stats in https://github.com/prometheus/procfs/blob/master/proc_stat.go). An alternative would be more verbose names like `CharsRead`, `CharsWritten`, etc., but I guess that would actually be inconsistent with https://github.com/prometheus/procfs/blob/master/proc_stat.go.